Intersect.Editor/Core/Graphics.cs

```diff
@@ -1514,7 +1524,7 @@ bool alternate
 
                     if (animation != null)
                     {
-                        float xpos = x * Options.Instance.Map.TileWidth + xoffset + Options.Instance.Map.TileWidth / 2;
+                        float xpos = x * tileWidth + xoffset + tileWidth / 2;
                         float ypos = y * Options.Instance.Map.TileHeight + yoffset + Options.Instance.Map.TileHeight / 2;
```

Comment:
By the way both of these lines can be written as `(base + 0.5f) * multiplier + offset`